sql_generators/mobile_kpi_support_metrics/templates/engagement.query.sql

```diff
@@ -1,4 +1,24 @@
 {{ header }}
+WITH device_manufacturer_counts AS (
+  SELECT
+    submission_date,
+    device_manufacturer,
+    COUNT(*) AS device_manufacturer_count,
+  FROM
+  `{{ project_id }}.{{ dataset }}.engagement_clients`
+  WHERE
+    {% raw %}
+    {% if is_init() %}
+      submission_date < CURRENT_DATE
```

Comment:
is this the condition that will run through when we backfill the table?
It seems we'll have vastly different criteria for the first run vs the rest as many smaller manufacturers could pass the threshold of 2k if we look at all historical data